content/07.pathogenesis.md

```diff
@@ -438,94 +419,42 @@ Several studies used phylogenetic analysis to determine the source of local COVI
-Additionally, ethnic minorities in the United Kingdom also tend to be younger than white British living in the same areas, yet the burden of COVID-19 is still more serious for minorities, especially people of Black Caribbean ancestry, both in absolute numbers and when controlling for age and location [@url:https://www.ifs.org.uk/inequality/chapter/are-some-ethnic-groups-more-vulnerable-to-covid-19-than-others/].
-Furthermore, the groups in the United States and United Kingdom that have been identified as carrying elevated COVID-19 burden, namely Black American, indigenous American, and Black and South Asian British, are quite distinct in their position on the human ancestral tree.
-What is shared across these groups is instead a history of disenfranchisement under colonialism and ongoing systematic racism.
-A large analysis of over 11,000 COVID-19 patients hospitalized in 92 hospitals across U.S. states revealed that Black patients were younger, more often female, more likely to be on Medicaid, more likely to have comorbidities, and came from neighborhoods identified as more economically deprived than white patients [@doi:10.1001/jamanetworkopen.2020.18039].
-This study reported that when these factors were accounted for, the differences in mortality between Black and white patients were no longer significant.
-Thus, the current evidence suggests that the apparent correlations between ancestry and health outcomes must be examined in the appropriate social context.
-
 ### Conclusions
 
+The novel coronavirus SARS-CoV-2 is the third HCoV to emerge in the 21st century, and research into previous HCoVs has provided a strong foundation for characterizing the pathogenesis and transmission of SARS-CoV-2.
+Critical insights into how the virus interacts with human cells have been gained from previous research in HCoV and other viral infections.
 As with other HCoV, the immune response to SARS-CoV-2 is likely driven by detection of its spike protein, which allows it to enter cells through the ACE2 receptor.
 Epithelial cells have also emerged as the major cellular target of the virus, contextualizing the respiratory and gastrointestinal symptoms that are frequently observed in COVID-19.
+Many of the mechanisms that facilitate the pathogenesis of SARS-CoV-2 are currently under consideration as possible targets for the treatment or prevention of COVID-19 [cite therapeutics].
+Research in other viruses also provides a foundation for understanding the transmission of SARS-CoV-2 among people and can therefore inform efforts to control the virus's spread.
+While the extent to which aerosol and fomite transmission play a role in the spread of SARS-CoV-2, in general this virus seems to be spread primarily by droplet transmission, much like SARS-CoV-1 and MERS-CoV.
```

Comment:
I was reading the transmission section and this conclusion. I think there is a lot less emphasis these days on fomite transmission and much more emphasis on transmission by respiratory droplets within close contact. Certainly fomite transmission does occur, but the CDC does not acknowledge it as the main route. I would have edited, but I don’t know who wrote this section so they may want to review it themselves.

https://www.cdc.gov/coronavirus/2019-ncov/more/scientific-brief-sars-cov-2.html
https://www.cdc.gov/coronavirus/2019-ncov/faq.html#Spread